### PR TITLE
move binding duck to v1

### PR DESCRIPTION
**Pull request by capri-xiyue**
-  Creating the v1 Binding duck (equal to v1beta1)

### CODE REVIEW

**Comment by capri-xiyue** [comment]
The codecov/project fails because there is no UT for biding_types.go. But I checked the code, it only defines some struct and implement some interface. There is no need to write UT for it. And we don't have UT for biding_types.go for both v1beta1 and v1alpha1.

**Comment by capri-xiyue** [comment]
assign @n3wscott

**Review by n3wscott — COMMENTED** [review]
/lgtm
/approve

**Comment by n3wscott** [comment]
/cc @mattmoor 

Matt to approve

File: apis/duck/v1/biding_types.go [hunk 1]
```diff
@@ -0,0 +1,92 @@
+/*
```
Comment (mattmoor) [inline apis/duck/v1/biding_types.go:1]: The file name is missing an `n`

Comment (capri-xiyue) [reply apis/duck/v1/biding_types.go]: done

**Review by mattmoor — APPROVED** [review]
/lgtm

**Pull request merged**